installer/PowerToysSetupCustomActions/PowerToysSetupCustomActions.vcxproj

```diff
@@ -43,6 +43,12 @@
     <LinkIncremental>false</LinkIncremental>
     <IncludePath>..\..\src\common\Telemetry;$(IncludePath)</IncludePath>
   </PropertyGroup>
+  <ItemDefinitionGroup>
+    <PreBuildEvent>
+    <Command>pwsh.exe -File parseRuntimes.ps1 -depsjsonpath "$(ProjectDir)..\..\$(Platform)\$(Configuration)\PowerToys.ColorPickerUI.deps.json" -depsfileslistspath "$(ProjectDir)DepsFilesLists.h" -productwxspath "$(ProjectDir)..\PowerToysSetup\Product.wxs"</Command>
```

Comment:
@snickler 
The script updates the original file. After the first execution the placeholders are replaced forever. Is this the expected behavior? I can imagine that it isn't and will lead to problems.

Reply:
Yeah, you're right... I'm trying to think of a non-intrusive way for this that only affects the build output.

I'm thinking:

- Copy those files to a backup location
- Make the modifications and compile
- On Post Build, overwrite with the original unmodified?

Reply:
@htcfreek - Updated per my thoughts above.

Reply:
@stefansjfw

Reply:
Oops, looks like it didn't like my modifications, but works locally? Hmm. I'll get this fixed.

Reply:
AH I know why.... This is something I didn't account for. The component GUID.